src/python/module/smartredis/client.py

```diff
@@ -121,7 +121,7 @@ def __deprecated_construction(self, cluster, address=None, logger_name="Default"
         except (PybindRedisReplyError, RuntimeError) as e:
             raise RedisConnectionError(str(e)) from None
 
-    def __new_construction(self, config_options=None, logger_name="Default"):  # pylint: disable=no-self-use
+    def __standard_construction(self, config_options=None, logger_name="Default"):  # pylint: disable=no-self-use
```

Comment:
Not directly related this PR, but thought I should bring it up while we are here:

You should be able to remove the `# pylint: disable=...` comment by simply decorating this method with `@staticmethod` and removing the `self` parameter

Reply:
Done